libcaf_core/caf/local_actor.hpp

```diff
@@ -264,6 +266,21 @@ class CAF_CORE_EXPORT local_actor : public abstract_actor {
     monitor(actor_cast<abstract_actor*>(whom), P);
   }
 
+  /// Adds a unidirectional `monitor` to `whom` with custom default handler.
+  /// @returns a disposable object that demonitors the actor on disposing.
+  /// @note This overload does not work with the @ref demonitor member function.
+  template <typename Handle, typename Fn>
+  disposable monitor(Handle whom, Fn func) {
+    using impl_t = detail::monitor_action<Fn>;
+    auto monitor_task = make_counted<impl_t>(std::move(func));
```

Comment:
A task is something that we actively schedule. Actions are usually triggered by something, so I'd use a name like `on_down`.